packages/env/README.md

```diff
@@ -507,6 +507,16 @@ On the development instance, `cwd` will be mapped as a plugin, `one-theme` will
 
 On the tests instance, `cwd` is still mapped as a plugin, but no theme is mapped. Additionally, while KEY_2 is still set to false, KEY_1 is overridden and set to false. 3000 overrides the default port as well.
 
+Another example would be serving the development site over a different URL, like this:
```

Comment:
Could you move this to the "examples" section somewhere below line 547 and include a heading? That should keep the file a bit more consistent :)

Reply:
How did I miss that there was an "examples" section!? Thank you.

Reply:
Something I'd love to understand better and possibly note in the docs is _how_ to look and/or _where_ to find keys (`DOCKER_REQUEST_URL` for example) that can be used for various goals in the env.

Reply:
So the `config` option is just letting you write a PHP constant to `wp-config.php`. So the constants are whatever WordPress allows (e.g. https://wpengineer.com/2382/wordpress-constants-overview/)

`DOCKER_REQUEST_URL` is not one of those constants, though -- as far as I can tell, it was only ever suggested here (#23947) as a way to enable this feature

Reply:
> So my big question with this PR -- does this actually work? It does not work for me locally, and as far as I can tell the issue requesting the feature (#23947) hasn't been implemented (yet). 🤔

It works for me. I'm serving a site over `mzmbo.test:8888` I'm interested in #23947 because I want to test some Oauth features which require an `https` callback url. In terms of the domain name, though, as I say, working for me. I don't know, yet, if serving over port `80` would make it so that I don't need to include the port in the browser url.

Reply:
did you have to set anything up in your /etc/hosts file?